operator-common/src/main/java/io/strimzi/operator/cluster/model/KafkaVersionTestConstants.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+public class KafkaVersionTestConstants {
```

Comment:
I'm not convinced this is a good idea. These are semi-artificial data used by some specific tests in the CO. They are designed for the tests which use them for upgrade testing etc. The do not necessarily need to correspond to what is in the actual cluster. For example as you can see, they are missing the 2.8.1 release. In different situations they will be missing other versions as well. It seems better to me to keep these as they were. 

For the TO tests, is there any really any reason to not load the data from the actual versions file?